core/ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCard.kt

```diff
@@ -156,12 +156,17 @@ fun NewsResourceAuthors(
     authors: List<Author>
 ) {
     if (authors.isNotEmpty()) {
-        // Only display first author for now
-        val author = authors[0]
+        // display all authors
+        val authorName = StringBuilder()
+        authors.forEachIndexed { index, author ->
```

Comment:
Thank you for your contribution Saeed. Could you replace this logic with `joinToString` method? I believe it's more suitable here.

@dturner Are we fine with displaying more authors?

Reply:
Of course

Reply:
Done